src/plugins/recordTypes/collectionobject/forms/default.jsx

```diff
@@ -63,6 +63,13 @@ const template = (configContext) => {
           <Field name="computedCrate" subpath="ns2:collectionobjects_anthropology" />
         </Row>
 
+        <Field name="objectClassList" subpath="ns2:collectionobjects_pahma">
```

Comment:
Make this `objectClassGroupList` to be consistent with naming of other repeating groups.